### ISSUE TITLE
oauth2client should depends on uritemplate

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Dragon-Born/google-api-python-client#247]
```
because gce.py relies on it
```

Original issue reported on code.google.com by `pro...@google.com` on 12 Mar 2013 at 6:35

**Comment by GoogleCodeExporter** [comment]
Original comment by `pro...@google.com` on 12 Mar 2013 at 6:35
- Added labels: **Component-oauth2client**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 1a9828bf14c5.
```

Original comment by `jcgregorio@google.com` on 13 Mar 2013 at 1:30
- Changed state: **Fixed**
